app/domain/people_skills_filter.rb

```diff
@@ -23,4 +24,8 @@ def filter_by_rated
     end
     entries
   end
+
+  def filter_by_level(entries)
+    level.nil? ? entries : entries.where('level >= ?', level)
+  end
```

Comment:
I would write it like this:
```
 def filter_by_level(entries)
    return entries if level.nil?

    entries.where('level >= ?', level)
  end
```
Sure you write more lines but it is more readable this way. In this ternary you either return the value of a variable or the result of a db query. Those are completely different "actions" if you get what I mean. It's not really one or the other (for which you would use a ternary) but a completely different thing.
It's not always about the length of the method. You need to make your code as readable as possible. If you combine the readability with ruby/rails methods, naming and CoC for smart, short, readable code, you're on top!